README.md

```diff
@@ -1 +1,46 @@
-# whatsapp-groupadd
\ No newline at end of file
+# Whatsapp-groupadd
+
+Whatsapp-groupadd is a Python code which helps it users to add people to a specific whatsapp group without directly saving their numbers.
+

```

Comment:
**WhatsApp** is a proper noun. It should be capitalized always.
Also, it looks odd when you write _whatsapp, Whatsapp_ etc.

- WhatsApp :white_check_mark:
- whatsapp ❌ 
- Whatsapp ❌